plugins/modules/elastic_agentlist_info.py

```diff
@@ -34,7 +60,8 @@ def main():
         port=dict(type='int', default=9243),
         username=dict(type='str', required=True),
         password=dict(type='str', no_log=True, required=True),   
-        verify_ssl_cert=dict(type='bool', default=True)
+        verify_ssl_cert=dict(type='bool', default=True),
+        deployment_info=dict(type='dict', default=None)
```

Comment:
We really need to get a black formatting pass added as part of our github actions on commit to any branch, but in the meantime, can you get it installed locally? otherwise we end up with weird formatting changes any time anyone else works on the code and it does things like add trailing commas